scripts/bench/ingest_and_snapshot.sh

```
@@ -0,0 +1,93 @@
+        sleep 1; attempt=$((attempt + 1))
+        [ $((attempt % 10)) -eq 0 ] && echo -e "${YELLOW}  Still waiting... (${attempt}s)${NC}"
+    done
+    echo -e "${RED}ERROR: API not ready after ${max}s${NC}"
+    return 1
+}
+
+echo -e "${BLUE}=== Ingest & Snapshot: ${BENCH_NAME} ===${NC}"
+
+# 1. Start services with benchmark-friendly env
+echo -e "${BLUE}Starting services...${NC}"
+cd "$REPO_ROOT"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)